src/infrastructure/arranging/sequencing.rs

```diff
@@ -185,23 +185,10 @@ mod tests {
         ( [$( $x:expr ),*], [$( $y:expr),*], $stream_id:expr) => {
             {
                 // initialize vector of given range on the left.
-                let mut before: Vec<u16> = Vec::new();
-                $(
-                    before.push($x);
-                )*
+                let before = [$($x,)*];
 
                 // initialize vector of given range on the right.
-                let mut after: Vec<u16> = Vec::new();
-                $(
-                    after.push($y);
-                )*
-
-                // generate test packets
-                let mut packets = Vec::new();
-
-                for (_, v) in before.iter().enumerate() {
-                    packets.push(Packet::new(*v, $stream_id));
-                }
```

Comment:
Same here

Reply:
Same principles applied here, as in `src/infrastructure/arranging/ordering.rs`